### PR TITLE
docs: add blog

### PR DESCRIPTION
**Pull request by RedJue**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 6307d2f</samp>

This pull request adds two new blog posts to the `docs/blog` folder, one in English and one in Chinese. The blog posts introduce the `ColorPicker` component, a new feature of Ant Design, and explain its implementation details and usage. The blog posts aim to share the development insights and showcase the new functionality.

### <samp>🤖 Generated by Copilot at 6307d2f</samp>

* Add new blog posts to introduce the `ColorPicker` component and its implementation ([link](https://github.com/ant-design/ant-design/pull/44698/files?diff=unified&w=0#diff-d35327ba9ca95fc92311e25845e955f36653c7621dd09dfad8581c913c6d73daR1-R113), [link](https://github.com/ant-design/ant-design/pull/44698/files?diff=unified&w=0#diff-a7fc50163332951384794ce6f1f7038704d241491d1179336aebfc3b153d9c06R1-R113))

### CODE REVIEW

**Comment by Yuiai01** [comment]
学习了 ❤️

File: docs/blog/color-picker.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: 实现一个颜色选择器
+date: 2023-09-07
+author: Redjue
+---
+
+大家好，我是 [Redjue（红果汁）](https://github.com/Redjue)，很荣幸今年有机会向 **Ant Design** 贡献了颜色选择器组件 `ColorPicker`，这是一次很棒的学习机会，也让我对 **Ant Design** 的开发流程有了更深入的了解。在这篇文章中，我将分享一下 `ColorPicker` 的实现过程。
```
Comment (afc163) [inline docs/blog/color-picker.zh-CN.md:7]: ColorPicker 可以直接给文档链接。

Comment (RedJue) [reply docs/blog/color-picker.zh-CN.md:7]: 我加一下

File: docs/blog/color-picker.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+title: 实现一个颜色选择器
```
Comment (kiner-tang) [inline docs/blog/color-picker.zh-CN.md:2]: 感觉我们花了很大的篇幅介绍色彩模型，不如标题改成这个？
```suggestion
title: 色彩模型与拾色器
```

Comment (RedJue) [reply docs/blog/color-picker.zh-CN.md:2]: 感觉可以，可以换一下

File: docs/blog/color-picker.en-US.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+---
+title: Color Models and Color Picker
+date: 2023-09-07
```
Comment (li-jia-nan) [inline docs/blog/color-picker.en-US.md:3]:
```suggestion
date: 2023-09-08
```

File: docs/blog/color-picker.zh-CN.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+title: 色彩模型与颜色选择器
+date: 2023-09-07
```
Comment (li-jia-nan) [inline docs/blog/color-picker.zh-CN.md:3]:
```suggestion
date: 2023-09-08
```

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**